internal/rbd/replicationcontrollerserver.go

```diff
@@ -52,6 +52,10 @@ const (
 	// another cluster
 	upAndStopped imageMirroringState = "up+stopped"
 
+	// If the state is up+unknown means the rbd-mirror daemon is
```

Comment:
nit: If the state is up+unknown it means that the rbd-mirror daemon is running and the image is demoted on both the clusters.